tests/core/test_tagged_data.c

```diff
@@ -9,25 +9,28 @@
 #include "crypto/iota_crypto.h"
 #include "unity/unity.h"
 
+#define TAG_LEN 64
```

Comment:
as the string in line#14, the length is 65, not 64.

Reply:
Removed "!" from the string in line#14 as max possible TAG_LEN is 64.